src/views/portfolio/projects/ProjectDetailsModal.vue

```diff
@@ -174,6 +187,50 @@
         }).catch((error) => {
           this.$toastr.w(this.$t('condition.unsuccessful_action'));
         });
+      },
+      retrieveParents: function() {
```

Comment:
Das kostet performance - was genau wird hier erreicht?

Reply:
Hier werden alle möglichen Parent-Projekte für ein Projekt mit einem API-Call geholt und so gefiltert, dass das jetzige Projekt und alle Kinder des jetzigen Projekts von der Selection ausgeschlossen sind. Zudem wird der Wert des "Select Parent"-Feld bei der Initialisierung auf den Wert des Parent vom Projekt gesetzt.

Die Funktion ist der schon vorhandenen Funktion `retrieveLicences()` in `ProjectCreateProjectModal.vue` nachempfunden:
https://github.com/rbt-mm/frontend/blob/94c10dcc8bc9a8b7da2f4fbf8037355c8c4d0e7f/src/views/portfolio/projects/ProjectCreateProjectModal.vue#L188-L201